src/forms/FormLines.tsx

```diff
@@ -37,7 +38,15 @@ export function FormLines(props: FormLinesProps) {
           "& > *": Css.mb2.$,
         }}
       >
-        {children}
+        {Children.map(children, (child) => {
+          if (child && typeof child === "object" && "type" in child && (child.type as any).isFormHeading) {
+            const clone = cloneElement(child, { isFirst: firstFormHeading });
+            firstFormHeading = false;
+            return clone;
```

Comment:
This is kind of sneaky. Though, this will only work if <FormHeading /> is a direct child of the <FormLines />,.... just playing devils advocate, this would be a problem someone were to use it like this:
```
<FormLines>
  <Tooltip title="Some extra info regarding the first FormHeading">
    <FormHeading />
  </Tooltip>
  <FormLine />
  <FormHeader /> (this one would be tagged as `isFirst`)
  ...
</FormLines>
```

Reply:
Right, agreed on the must-be-direct-child con...

I was thinking, that con probably means we shouldn't use this for `ModalFooter` things b/c you'd have to have immediate children like:

* Modal
  * ModalHeader
  * ModalFooter

And I think, well especially for superdrawer, we want actions/footers to be nested down in the react tree somewhere like:

* MyDrawerPaage
  * Tab2
    * SuperDrawerActions

...so, yeah, can't use this approach in those sort of places, but will probably try it here and see how it goes.